index.js

```diff
@@ -163,11 +165,19 @@ module.exports = {
       addGroupTangle(content, (err, content) => {
         if (err) return cb(err)
 
-        ssb.db.create({ content, encryptionFormat: 'box2' }, (err, msg) => {
-          if (err) return cb(err)
+        ssb.db.create(
+          {
+            content,
+            encryptionFormat: 'box2',
+            feedFormat: 'buttwoo-v1',
```

Comment:
If we don't use `buttwoo-v1` in loadOrCreate, then I can see that this line wouldn't work. In fact, what we need here is metafeeds, so we can create a subfeed (under a shard) which has the buttwoo feedFormat, and then pass the keys to that buttwoo identity to ssb.db.create().

I also have a meta question here, which is: why buttwoo for groups?

Reply:
my understanding was that buttwoo would be used for new feeds and that groups would only be on new feeds. but i guess we'll discuss this in an upcoming meeting.

this line does feel a bit weird. why do i have to input the feed format into create? couldn't create detect what feed format is used on the feed i'm posting to and simply use that format? or are feeds supposed to be able to contain multiple formats?